sql/clients_last_seen_raw_v1.sql

```diff
@@ -87,11 +89,11 @@ WITH
       1 AS bits
     FROM
       UNNEST(experiments.key_value)) AS days_seen_in_experiment,
-    * EXCEPT (submission_date_s3)
+    * EXCEPT (submission_date)
```

Comment:
include the change here too for CI
```suggestion
    * EXCEPT (submission_date) REPLACE (CAST(sample_id AS STRING) AS sample_id)
```